starter-code/scripts/models/article.js

```diff
@@ -41,28 +49,33 @@
         (most recent article first!), and then hand off control to the View.
       Otherwise (if the DB is empty) we need to retrieve the JSON and process it. */
 
-    webDB.execute('', function(rows) { // TODO: fill these quotes to query our table.
+    webDB.execute([{'sql': 'SELECT * FROM articles_table'}], function(rows) { // TODO: fill these quotes to query our table.
```

Comment:
You're supposed to retrieve the most recent article first. This will just return the articles in the order you put them in.